restic/internal/backup_test.py

```diff
@@ -109,6 +109,38 @@ def test_excludes_single_exclude_file(self, mock_execute):
             'bad-songs.txt'
```

Comment:
Now that the `paths` parameter to `backup` is optional, we should update all the tests to call with `.backup(paths=[...])`

Because otherwise it's confusing that we pass `paths` as a positional and everything else as named.